### PR TITLE
🐛 fix: make redoc available for production environment

### LINKED ISSUE #353: [Good First Issues]: Make redoc available for production environment
# Description
We've integrated [Redoc](https://github.com/Redocly/redoc) for [our api project](https://github.com/featbit/featbit/tree/main/modules/back-end) at http://localhost:5000/docs/index.html, but now it's unavailable for production environment.

image

Here's the code that enables swagger, swagger ui and redoc. While swagger and swagger ui is limited to the development environment, redoc is enabled for all environments.

https://github.com/featbit/featbit/blob/23e93170fb1ab2baf2f8c4350c5ae91cdb351a12/modules/back-end/src/Api/Setup/MiddlewaresRegister.cs#L19-L50

The bug here is that Redoc has a dependency on Swagger, and therefore, Swagger must be enabled across all environments to make Redoc available in all environments.

# Scope
- Service: API
- File to change: modules/back-end/src/Api/Setup/MiddlewaresRegister.cs

# Setup dev environment

Please read the [doc](../blob/main/Development.md) to set up your development environment.

# Deadline

We usually give 2 weeks for a feature request, which means you have 2 weeks to implement the feature starting from the
day it is assigned to you. If you have any difficulties, please contact us
in [Slack](https://join.slack.com/t/featbit/shared_invite/zt-1ew5e2vbb-x6Apan1xZOaYMnFzqZkGNQ).

# How to implement

1. Fork the FeatBit repository.
2. Create a branch issues-{issues number}
3. Implement the task based on Description and Scope mentioned above.
4. Submit your PR

# Contribution points

Each contribution counts a certain number of points, depending on its difficulty. Contributors earn the appropriate
points when the work is merged. Contribution points are used to describe the contributions that contributors have made.
They can also be used to receive rewards for community events. You can view the current community submissions on a
public [google sheet](https://docs.google.com/spreadsheets/d/1ukyXgi_jRPeXj7EAST0IrnPfLOQ6xDBkcyAJY9N-Yb4/edit#gid=1117970540).

Points: 4

# How to claim to solve the issue

If you want to implement this function, please leave a comment in this issue like:

> I'd like to implement this function, please assign this issue to me.

### PR DESCRIPTION
**Pull request by defqd**
fixed #353

### CODE REVIEW

File: modules/back-end/src/Api/Setup/MiddlewaresRegister.cs [hunk 1]
```diff
@@ -16,10 +16,12 @@ public static WebApplication SetupMiddleware(this WebApplication app)
         // external use
         app.MapHealthChecks("health/liveness", new HealthCheckOptions { Predicate = _ => false });
 
-        // enable swagger in development environment
+        //enable swagger
```
Comment (deleteLater) [inline modules/back-end/src/Api/Setup/MiddlewaresRegister.cs]: nit: add a space after the double slash.

**Comment by deleteLater** [comment]
I updated the pr title.

File: modules/back-end/src/Api/Setup/MiddlewaresRegister.cs (hunk 1 above)
Comment (defqd) [reply modules/back-end/src/Api/Setup/MiddlewaresRegister.cs]: @deleteLater I pushed a commit please review

**Review by deleteLater — APPROVED** [review]
LGTM! Thanks for your work on this PR ❤️

**Pull request merged**